app/src/main/res/xml/preferences.xml

```diff
@@ -34,14 +34,4 @@
 
     </PreferenceCategory>
 
-    <PreferenceCategory
-        app:key="notifications_category"
-        app:title="Notifications">
-
-        <SwitchPreferenceCompat
-            app:key="switch_notifications"
-            app:title="Enable notifications" />
-
-    </PreferenceCategory>
-
```

Comment:
I did it because this setting already exists in the settings app of the phone, where you can choose how the notifications for this app are received (sound, vibration, ...) and which one to enable or disable (due to the notifications channels). It might be possible to implement it directly in the app but it should not be a priority in my opinion, as it would merely be a "shortcut" to avoid going in the settings app.

Reply:
Thank you for the answer